Native Instruments has some quirks with Drag N Drop. fix it.

Silence drag and drop FIXMEs - they cause too much spam with NI plugins. This
way, if I am launching from a terminal or debugging, I don't have to look at it.

--- wine-1.7.14/dlls/ole32/ole2.c.orig	2011-12-08 14:44:37.811271736 -0800
+++ wine-1.7.14/dlls/ole32/ole2.c	2011-12-08 14:45:46.934191902 -0800
@@ -2198,6 +2199,19 @@
     }
     case WM_TIMER:
     case WM_MOUSEMOVE:
+    {
+      TrackerWindowInfo* p = (TrackerWindowInfo*)GetWindowLongPtrA(hwnd, 0); 
+      if (!p->trackingDone)
+      {
+       p->dwKeyState = OLEDD_GetButtonState();
+       if (p->dwKeyState & MK_LBUTTON)
+           OLEDD_TrackStateChange(p);
+       else
+           OLEDD_TrackStateChange(p);
+      }
+
+      break;
+    }
     case WM_LBUTTONUP:
     case WM_MBUTTONUP:
     case WM_RBUTTONUP:
--- a/dlls/shell32/dragdrophelper.c.orig	2020-06-01 17:30:37.800095568 -0400
+++ b/dlls/shell32/dragdrophelper.c	2020-06-01 17:30:28.605069400 -0400
@@ -89,7 +89,7 @@
 	TRACE ("-- Interface: (%p)->(%p)\n", ppvObj, *ppvObj);
 	return S_OK;
     }
-    FIXME ("-- Interface: E_NOINTERFACE\n");
+    //FIXME ("-- Interface: E_NOINTERFACE\n");
     return E_NOINTERFACE;
 }
 
@@ -126,35 +126,35 @@
 	DWORD dwEffect)
 {
     dragdrophelper *This = impl_from_IDropTargetHelper(iface);
-    FIXME ("(%p)->(%p %p %p 0x%08x)\n", This,hwndTarget, pDataObject, ppt, dwEffect);
+    //FIXME ("(%p)->(%p %p %p 0x%08x)\n", This,hwndTarget, pDataObject, ppt, dwEffect);
     return E_NOTIMPL;
 }
 
 static HRESULT WINAPI IDropTargetHelper_fnDragLeave (IDropTargetHelper * iface)
 {
     dragdrophelper *This = impl_from_IDropTargetHelper(iface);
-    FIXME ("(%p)->()\n", This);
+    //FIXME ("(%p)->()\n", This);
     return E_NOTIMPL;
 }
 
 static HRESULT WINAPI IDropTargetHelper_fnDragOver (IDropTargetHelper * iface, POINT* ppt, DWORD dwEffect)
 {
     dragdrophelper *This = impl_from_IDropTargetHelper(iface);
-    FIXME ("(%p)->(%p 0x%08x)\n", This, ppt, dwEffect);
+    //FIXME ("(%p)->(%p 0x%08x)\n", This, ppt, dwEffect);
     return E_NOTIMPL;
 }
 
 static HRESULT WINAPI IDropTargetHelper_fnDrop (IDropTargetHelper * iface, IDataObject* pDataObject, POINT* ppt, DWORD dwEffect)
 {
     dragdrophelper *This = impl_from_IDropTargetHelper(iface);
-    FIXME ("(%p)->(%p %p 0x%08x)\n", This, pDataObject, ppt, dwEffect);
+    //FIXME ("(%p)->(%p %p 0x%08x)\n", This, pDataObject, ppt, dwEffect);
     return E_NOTIMPL;
 }
 
 static HRESULT WINAPI IDropTargetHelper_fnShow (IDropTargetHelper * iface, BOOL fShow)
 {
     dragdrophelper *This = impl_from_IDropTargetHelper(iface);
-    FIXME ("(%p)->(%u)\n", This, fShow);
+    //FIXME ("(%p)->(%u)\n", This, fShow);
     return S_OK;
 }
 
@@ -193,7 +193,7 @@
 {
     dragdrophelper *This = impl_from_IDragSourceHelper(iface);
 
-    FIXME("(%p)->(%p, %p): stub\n", This, dragimage, object);
+    //FIXME("(%p)->(%p, %p): stub\n", This, dragimage, object);
 
     return E_NOTIMPL;
 }
@@ -203,7 +203,7 @@
 {
     dragdrophelper *This = impl_from_IDragSourceHelper(iface);
 
-    FIXME("(%p)->(%p, %s, %p): stub\n", This, hwnd, wine_dbgstr_point(pt), object);
+    //FIXME("(%p)->(%p, %s, %p): stub\n", This, hwnd, wine_dbgstr_point(pt), object);
 
     return E_NOTIMPL;
 }
